src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -477,7 +491,9 @@ public void deleteStudyIfNotCreationInProgress(UUID studyUuid, String userId) {
                         studyServerExecutionService.runAsync(() -> deleteEquipmentIndexes(deleteStudyInfos.getNetworkUuid())),
                         studyServerExecutionService.runAsync(() -> networkStoreService.deleteNetwork(deleteStudyInfos.getNetworkUuid())),
                         studyServerExecutionService.runAsync(deleteStudyInfos.getCaseUuid() != null ? () -> caseService.deleteCase(deleteStudyInfos.getCaseUuid()) : () -> {
-                        })
+                        }),
```

Comment:
move it above next to the others that do the same kind of thing just for clarity

Reply:
Done